test/e2e/imagebundle/push_bundle_test.go

```diff
@@ -340,38 +340,38 @@ var _ = Describe("Push Bundle", func() {
 				{
 					OS:           "linux",
 					Architecture: "amd64",
-				}: "sha256:f0657165bbbc518bd1adb6a3605532c0c30b237ee1a85ce51a94ab78878c996e",
+				}: "sha256:f6aa16be680b2cb3cae9bb2043b70447f705b6047a97ffa4cde9b3ac7fdae545",
```

Comment:
These SHAs changed upstream?

Reply:
Yep and I think it got pushed again even after these tests passed 🤦 
https://hub.docker.com/layers/library/busybox/1.37.0-musl/images/sha256-e6030835df7fb6514e08929009cef5175557237805dd62cdb67db94201f8af09 

>Last pushed
about 4 hours by [doijanky](https://hub.docker.com/u/doijanky)